spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkScanBuilder.java

```diff
@@ -105,6 +108,7 @@ public Filter[] pushFilters(Filter[] filters) {
         expr = SparkFilters.convert(filter);
       } catch (IllegalArgumentException e) {
         // converting to Iceberg Expression failed, so this expression cannot be pushed down
+        LOG.warn("converting to Iceberg Expression failed, so this expression cannot be pushed down.", e);
```

Comment:
Sorry to be nitty, but capitlization does not seem right.  What do think like 'Failed to convert filter to Iceberg expression, skipping push down").  

And I wonder, does it help to add the filter to the message?

Reply:
+1 to adding the filter to the message.

So here, I would think maybe LOG.warn(" 'Failed to convert filter to Iceberg expression, skipping push down for this expression: {}", filter, e);'

Reply:
Sounds good! Changed.